contracts/src/TeleporterTokenDestination.sol

```diff
@@ -488,12 +511,29 @@ abstract contract TeleporterTokenDestination is
         // Deposit the funds sent from the user to the bridge,
         // and set to adjusted amount after deposit
         amount = _deposit(amount);
+
+        // The bridged amount must cover the secondary fee, because the secondary fee
+        // is directly subtracted from the bridged amount on the intermediate chain
+        // performing the multi-hop, before forwarding to the final destination chain.
         require(
-            amount > primaryFee + secondaryFee,
-            "TeleporterTokenDestination: insufficient amount to cover fees"
+            amount > secondaryFee, "TeleporterTokenDestination: insufficient amount to cover fees"
```

Comment:
Should we check that the scaled amount is greater than the scaled secondary fee amount? As is, I think it's possible for them to be equal due to rounding.

Reply:
Combined this check and the scaled tokens > 0 check below